llm/deepseek-r1/README.md

```diff
@@ -0,0 +1,259 @@
+        "tool_calls": []
+      },
+      "logprobs": null,
+      "finish_reason": "stop",
+      "stop_reason": null
+    }
+  ],
+  "usage": {
+    "prompt_tokens": 15,
+    "total_tokens": 985,
+    "completion_tokens": 970,
+    "prompt_tokens_details": null
+  },
+```
```

Comment:
i think the final two lines (prompt_logprobs and the closing bracket) are missing